src/commands/dev/mod.rs

```diff
@@ -46,12 +47,32 @@ pub async fn dev(
         let server_config = server_config.clone();
         async move {
             Ok::<_, failure::Error>(service_fn(move |req| {
```

Comment:
i think this closure deserves to either be a well named function in its own right, or to have a comment above.

Reply:
i don't think i can do that since hyper doesn't re-export the type `MakeServiceFn` from what I can tell. i've simplified the closure as much as i can by pulling some if it out into a separate function though